astrokat/__init__.py

```diff
@@ -2,6 +2,16 @@
 from __future__ import division
 from __future__ import absolute_import
 
+# Constants and defaults
+_DEFAULT_LEAD_TIME = 3.0  # lead time [sec]
```

Comment:
3 seconds is a mean time, what has been found is that sometimes it can be as large as 5 seconds
Which is why the lead time has been accommodated (temporarily) in the yaml configuration file to be set by the user

The system awaits the Commissioning to determine optimal values

Reply:
OK, good point. For the checking we could gather all delaycal data as a good sample when the noise diodes actually activated. Good luck finding a happy smiling head doing that though...